packages/prompts/src/index.ts

```diff
@@ -409,6 +414,27 @@ export const outro = (message = '') => {
 	process.stdout.write(`${color.gray(S_BAR)}\n${color.gray(S_BAR_END)}  ${message}\n\n`);
 };
 
+export type LogMessageOptions = {
+	symbol?: string;
+};
+export const log = {
+	message: (message = '', { symbol = color.gray(S_BAR) }: LogMessageOptions = {}) => {
```

Comment:
**Note:** message is optional, so you can use this to add extra vertical spacing (newlines).

To make it more explicit, we could add another convenience function:

```ts
log.newline = () => {
  log.message();
}
```

Reply:
I don't think an extra method is necessary, `log.message()` works just like `console.log()` in this case!

Reply:
OK, should we maybe skip a newline in the empty case? I.e. just output one bar?

Reply:
I think that makes sense, yes